libs/core/src/lib/message-box/message-box-semantic-icon/message-box-semantic-icon.component.ts

```diff
@@ -40,15 +35,15 @@ export class MessageBoxSemanticIconComponent {
     get _semanticIcon(): string {
         switch (this.messageBoxConfig.type) {
```

Comment:
As a side note, we can transfer this switch to object lookup and then we could make it configurable by user